Lib/test/test_typing.py

```diff
@@ -4310,6 +4310,11 @@ def test_annotated_in_other_types(self):
         X = List[Annotated[T, 5]]
         self.assertEqual(X[int], List[Annotated[int, 5]])
 
+    def test_annotated_mro(self):
```

Comment:
This looks unrelated to your PR title, any idea what happened?

Reply:
Looks like I backported a different commit 🤔 

Thanks for noticing!

Reply:
I got the same result second time: https://github.com/python/cpython/pull/30707